api/core/helper/code_executor/code_executor.py

```diff
@@ -18,8 +18,8 @@
 logger = logging.getLogger(__name__)
 
 # Code Executor
-CODE_EXECUTION_ENDPOINT = get_env('CODE_EXECUTION_ENDPOINT')
-CODE_EXECUTION_API_KEY = get_env('CODE_EXECUTION_API_KEY')
+CODE_EXECUTION_ENDPOINT = os.environ.get('CODE_EXECUTION_ENDPOINT', 'http://sandbox:8194')
```

Comment:
Using `current_app.config.get()` would be better

Reply:
I have tried to use `current_app.config.get()` at first. However, it blocks the unit tests, as no Flask app context at the runtime.